modules/microshift-configuring-hosts-for-mirror.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-configuring-hosts-for-mirror_{context}"]
+= Configuring hosts for mirror registry access
+
+To configure a {product-title} host to use a mirror registry, you must give the host access and create a configuration file with the host name and port through which the registry can be reached.
+
+.Prerequisites
+* Your mirror host has access to the internet.
```

Comment:
Mirror is air-gapped most of the times.

Reply:
Did you mean here that MicroShift host should have network access to the mirror?

Reply:
I was looking at, "Download Images on a host with the Internet access." https://docs.google.com/document/d/1aqgjxP2YqIGi2djU96vN2yykDZpIC8s2hg4QHTCz390/edit#heading=h.pawolv522s21

Reply:
OK, I was confused by names. To reiterate, we have 3 entities involved
- mirror host - with Internet access
- mirror registry - no Internet access
- microshift host - access to mirror registry
Looks like the above prerequisite is OK.